docs/src/whatsnew/3.14.rst

```diff
@@ -1,30 +1,50 @@
 .. include:: ../common_links.inc
 
-|iris_version| |build_date| [unreleased]
-****************************************
+v3.14 (31 Oct 2025 [release candidate])
+***************************************
 
 This document explains the changes made to Iris for this release
 (:doc:`View all changes <index>`.)
 
 
-.. dropdown:: |iris_version| Release Highlights
+.. dropdown:: 3.14 Release Highlights
    :color: primary
    :icon: info
    :animate: fade-in
    :open:
 
-   The highlights for this major/minor release of Iris include:
+   The highlights for this minor release of Iris include:
 
-   * N/A
+   * Added a convenience function :func:`~iris.util.make_gridcube` for creating
+     a 2D cube with a grid of a specified shape. This is useful for functions
```

Comment:
> a grid of a specified shape

I think it would be better to say "a specified grid", since I guess users will almost always want to specify the actual X and Y points.